src/tools/MacOS Installation.md

```diff
@@ -0,0 +1,47 @@
+# MacOS Installation
+
+### 1. Download
+
+Download SlimeVR for Mac (the .dmg file from (https://github.com/SlimeVR/SlimeVR-Server/releases/latest/download/SlimeVR-mac.dmg)
+
+### 2. Install
+
+Open the .dmg and drag SlimeVR app into Applications.
+
+### 3. Open
+
+Open SlimeVR from Applications. (You will get this error shown below)
+
+
+<img width="268" alt="warning" src="https://github.com/user-attachments/assets/018db046-56bc-4b74-9104-5a92e8d77408" />
```

Comment:
All these images should be added to the docs resources and linked from the docs site itself rather than a hard link to GitHub. See how images are done in [./src/quick-setup.md](https://github.com/SlimeVR/SlimeVR-Docs-Site/blob/main/src/quick-setup.md) for reference.